helper/include/odp/helper/strcpy.h

```diff
@@ -0,0 +1,62 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2024 Nokia
+ */
+
+/**
+ * @file
+ *
+ * ODP string copy helper
+ */
+
+#ifndef ODPH_STRCPY_H_
+#define ODPH_STRCPY_H_
+
+#include <odp/api/hints.h>
+#include <string.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup odph_strcpy ODPH strcpy
+ *
+ * ODP string copy helper
+ *
+ * @{
+ */
+
+/**
+ * Copy a string
+ *
+ * Like strncpy(), but additionally ensure that the destination string is NUL-terminated, unless sz
```

Comment:
Typo: `NUL` and I'd perhaps change `ensure` -> `ensures` and `return` -> `returns`.

Reply:
NUL is actually correct here.

Reply:
Oh, learned something new today 👍

Reply:
However, the references in https://en.wikipedia.org/wiki/Null_character mostly use "null character" and "null terminated", etc. So I'm doing the same in v2.

Reply:
Yes, "null character" is also used in C standard, so null-terminated would probably be better than NUL-terminated. But NULL would not be right, I think.